lib/main_common.pm

```diff
@@ -86,6 +88,13 @@ sub setup_env {
     }
 }
 
+sub any_desktop_is_applicable() {
+    return get_var("DESKTOP") !~ /textmode/;
+}
+
+sub console_is_applicable() {
```

Comment:
1. see answer to previous comment 
2. also discuss this with @okurz  , so probably he can add his 5 cents here . Oli what do you think should we rename it ? 
And again even if we need to rename it I think it should be different PR to not create a mess in git log